src/modules/ShortcutGuide/ShortcutGuide.Ui/Strings/en-us/Resources.resw

```diff
@@ -117,55 +117,105 @@
+  </data>
+  <data name="Overview" xml:space="preserve">
+    <value>Overview</value>
+    <comment>Category title</comment>
+  </data>
+  <data name="PinnedListTitle.Text" xml:space="preserve">
+    <value>Pinned Shortcuts</value>
+    <comment>Shortcuts refers to keyboard shortcuts</comment>
+  </data>
+  <data name="PinShortcut" xml:space="preserve">
+    <value>Pin</value>
+    <comment>Refers to the action of pinning something on a pin board</comment>
+  </data>
+  <data name="RecommendedListTitle.Text" xml:space="preserve">
+    <value>Recommended Shortcuts</value>
+    <comment>Shortcuts refers to keyboard shortcuts</comment>
   </data>
-  <data name="No_Action" xml:space="preserve">
-    <value>No action</value>
+  <data name="SearchBlank" xml:space="preserve">
+    <value>No results found</value>
+    <comment>Results refers to search results</comment>
   </data>
-  <data name="Restore" xml:space="preserve">
-    <value>Restore</value>
+  <data name="SearchBox.PlaceholderText" xml:space="preserve">
+    <value>Search shortcuts</value>
   </data>
-  <data name="Snap_Right" xml:space="preserve">
-    <value>Snap right</value>
+  <data name="SettingsButton.ToolTipService.ToolTip" xml:space="preserve">
+    <value>Open settings</value>
   </data>
-  <data name="Snap_Left" xml:space="preserve">
-    <value>Snap left</value>
+  <data name="ShowOOBE.Content" xml:space="preserve">
+    <value>Show welcome screen</value>
   </data>
-  <data name="Snap_Upper_Right" xml:space="preserve">
-    <value>Snap upper right</value>
+  <data name="TaskbarLaunchShortcutsDescription.Text" xml:space="preserve">
+    <value>Use these keys together with a number displayed below</value>
+    <comment>keys refers to keyboard keys</comment>
+  </data>
+  <data name="TaskbarLaunchShortcutsTitle.Text" xml:space="preserve">
+    <value>Taskbar Launch Shortcuts</value>
+    <comment>Shortcuts refers to keyboard shortcuts</comment>
+  </data>
+  <data name="Title" xml:space="preserve">
+    <value>Shortcut Guide</value>
+    <comment>Shortcut refers to a keyboard shortcut</comment>
+  </data>
+  <data name="UnpinShortcut" xml:space="preserve">
+    <value>Unpin</value>
+    <comment>Refers to the action of unpinning something from a pin board</comment>
+  </data>
+  <data name="OOBEDescription1.Text" xml:space="preserve">
+    <value>The new Shortcut Guide V2 now includes shortcuts from other applications (when they are running), the ability to pin your favorite shortcuts (by right-clicking on them) and a search function.
+Currently the following apps are available built in:</value>
+  </data>
+  <data name="OOBEDescription2.Text" xml:space="preserve">
+    <value>More apps are to be added soon!
+
+Click on a section in the header to see this apps shortcuts.</value>
   </data>
-  <data name="Snap_Upper_Left" xml:space="preserve">
-    <value>Snap upper left</value>
+  <data name="OOBEDescription1_1.Text" xml:space="preserve">
```

Comment:
Tip: using consecutive numbers is nice, until the order of things is changed...

Reply:
What would you suggest instead?